### PR TITLE
Fix/update spring

### PR DESCRIPTION
**Pull request by AndreiOla**
**Changed**
Updated Spring to 5.2.20.RELEASE, in order to fix critical vulnerability, for tag 2.8.3.7 .

### CODE REVIEW

**Comment by tlawrie** [comment]
Hi @AndreiOla this seems to have a number of other changes not related to the dependency change. Could you clarify?

**Comment by AndreiOla** [comment]
Hi @tlawrie I made changes starting from tag 2.8.3.7. And I want to build only those changes, on top of tag 2.8.3.7. I was just trying to figure out how can I do that, since git does not allow me to open a PR against a certain tag.

**Comment by tlawrie** [comment]
So typically you would create a branch from the tag i.e. 

```
git checkout <tag>
git checkout -b fix-<branch_name>
```

And then when you tag it, it would be the original <tag> and then either a fourth hot fix number like `.1` or `-fix.1` or something like that

**Comment by AndreiOla** [comment]
Thanks @tlawrie !
I made separate branches from the tags in my fork, made changes, and then used @amhudson branches in (https://github.com/boomerang-io/flow.service.workflow), for the tag, since I do not have sufficient rights to push into (https://github.com/boomerang-io/flow.service.workflow) directly.
That is correct, that we don't need to merge into main. Just build hot fixes for those tags.

**Comment by amhudson** [comment]
@tlawrie yes, that's correct

File: pom.xml [hunk 1]
```diff
@@ -31,8 +31,74 @@
 		<jacoco.version>0.8.2</jacoco.version>
 		<jsonwebtoken.version>0.10.7</jsonwebtoken.version>
 		<jackson.databind.version>2.9.9.3</jackson.databind.version>
+		<log4j2.version>2.17.0</log4j2.version>
+		<spring.version>5.2.20.RELEASE</spring.version>
 		<log4j2.version>2.17.2</log4j2.version>
```
Comment (amhudson) [inline pom.xml:36]: there are 2 `log4j2.version`, remove `2.17.0`.

**Review by amhudson — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (AndreiOla) [reply pom.xml]: Fixed.

**Review by amhudson — APPROVED** [review]

**Pull request merged**